README.md

```diff
@@ -1,9 +1,7 @@
 <p align="center">
   <a href="https://fingerprint.com">
     <picture>
```

Comment:
Do we still need the picture element?

Reply:
no idea tbh. Let me try without

Reply:
removed it, ty! Could you re-review pls? @JuroUhlar

Reply:
since there is only 1 image, it made sense to remove it